_posts/2022-09-30-release-table-store-0.2.1.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title:  "Apache Flink Table Store 0.2.1 Release Announcement"
+date: 2022-09-30T22:00:00.000Z
+categories: news
+authors:
+- JingsongLi:
+  name: "Jingsong Lee"
+
+excerpt: The Apache Flink Community is pleased to announce a bug fix release for Flink Table Store 0.2.
+
+---
+
+The Apache Flink Community is pleased to announce the first bug fix release of the Flink Table Store 0.2 series.
+
+This release includes 14 bug fixes, vulnerability fixes, and minor improvements for Flink Table Store 0.2.
+Below you will find a list of all bugfixes and improvements. For a complete list of all changes see:
+[JIRA](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12352257).
+
+We highly recommend all users upgrade to Flink Table Store 0.2.1.
+
+# Release Artifacts
+
+## Binaries
+
+You can find the binaries on the updated [Downloads page]({{ site.baseurl }}/downloads.html).
+
+# Release Notes
+
+<h2>        Bug
+</h2>
+<ul>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29098'>FLINK-29098</a>] -         StoreWriteOperator#prepareCommit should let logSinkFunction flush first before fetching offset
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29241'>FLINK-29241</a>] -         Can not overwrite from empty input
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29273'>FLINK-29273</a>] -         Page not enough Exception in SortBufferMemTable
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29278'>FLINK-29278</a>] -         BINARY type is not supported in table store
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29295'>FLINK-29295</a>] -         Clear RecordWriter slower to avoid causing frequent compaction conflicts
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29345'>FLINK-29345</a>] -         Too many open files in table store orc writer
```

Comment:
It seems the fix version of [FLINK-29345](https://issues.apache.org/jira/browse/FLINK-29345) is 0.3.0 only, please check and confirm whether this is included in 0.2.1